README.md

```diff
@@ -208,17 +171,12 @@ Example:
 
 > Mention all of the collaborators of this project.
 
-👤 **Author1**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
-
-👤 **Author2**
+👤 **Authors**
+Outkarm Johnson
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+- GitHub: [Outkarm](https://github.com/Outkarm)
+- GitHub: [Outkarm](https://github.com/Outkarm)
+- LinkedIn: [John Kpordje](https://www.linkedin.com/in/john-kpordje-866749241/)
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/224508480-93040c66-d709-4efa-9cd9-b16517cfefac.png)

**[OPTIONAL]**

- [ ] It would be nice if you add messages in both sections to make your readme look descriptive and professional 😊